extensions/xdebug/lib/XDebugUtil.php

```diff
@@ -13,10 +13,23 @@
 namespace PhpBench\Extensions\XDebug;
 
 use PhpBench\Executor\ExecutionContext;
+use RuntimeException;
 
 class XDebugUtil
 {
-    public static function filenameFromContext(ExecutionContext $context, $extension = ''): string
+    /**
+     * @var string|false
+     */
+    private $xdebugVersion;
+    private $xdebugUseCompression;
+
+    public function __construct($xdebugVersion, $xdebugUseCompression)
```

Comment:
missing type hints @param and whatever `$xdebugUseCompression` is, I think this directory is missing from the PHPStan paths.